GliaWidgetsObjC/README.md

```diff
@@ -0,0 +1,23 @@
+## Objective-C wrapper for [GliaWidgets](https://github.com/salemove/ios-sdk-widgets)
+
+GliaWidgetsSDK developed used pure Swift and doesn't provide Objective-C compatible interfaces. This sample Xcode project shows possibility to get full compatibility with Objective-C.
+
+##### NB! This project is not updating with latest released GliaWidgetsSDK versions and shows how to get Objective-C interfaces only.
+
+### Project
+
+Xcode project nests all GliaWidgetsSDK frameworks, introduces `GliaWidgetsWrapper.swift` that maps Swift interfaces to Objective-C.
+
+### How does it work?
+The `@objc` and `@objcMembers` attributes in Swift make Swift API available in Objective-C and the Objective-C runtime.
+
+### How to use
+
+To import and use `ios-sdk-widgets-objc` framework you need to:
+- update nested frameworks to latest versions. [See "Update Dependency"](#update-dependency)
+- build Xcode project to get `framework` in `Products` folder.
```

Comment:
Not 100% sure of this, but if you use "get", maybe it's "to folder" not "in folder"? But it depends what is actually happening here technically :) 
```suggestion
2. Build the Xcode project to get `framework` to the `Products` folder.
```